content/04.results.md

```diff
@@ -46,3 +46,38 @@ These results suggest that selecting target genes for mutation prediction based
 **B.** Overall distribution of performance across 3 gene sets. Each data point represents mean cross-validated AUPR difference, compared with a baseline model trained on shuffled mutation presence/absence labels, for one gene in the given gene set; notches show bootstrapped 95% confidence intervals. "random" = 107 random genes, "most mutated" = 107 most mutated genes, "Vogelstein et al." = 107 cancer related genes from Vogelstein et al. 2013 gene set.
 **C, D, E.** Volcano-like plots showing mutation presence/absence predictive performance for each gene in each of the 3 gene sets. The _x_-axis shows the difference in mean AUPR compared with a baseline model trained on shuffled labels, and the _y_-axis shows _p_-values for an unpaired _t_-test comparing cross-validated AUPR values.
 ](images/figure_2.png){#fig:expression_gene_sets width="90%"}
+
+### Comparing gene expression and DNA methylation reveals widespread redundancy
+
+We compared gene expression with DNA methylation as downstream readouts of the effects of cancer alterations.
+In these analyses, we considered both the 27K probe and 450K probe methylation datasets generated for the TCGA Pan-Cancer Atlas.
+Based on the results of the gene set experiments, we considered only the set of cancer-related genes derived from Vogelstein et al.
+We used samples that had data for each of the data types being compared, including somatic mutation data to generate mutation labels.
+This process retained 7,981 samples in the intersection of the expression, 27K methylation, 450K methylation, and mutation datasets, which we used for subsequent analyses.
+The most frequent missing data types were somatic mutation data (1,114 samples) and 450K methylation data (1,072 samples) (Figure {@fig:methylation}A).
+
+Overall performance across data types for 84 genes from is summarized in Figure {@fig:methylation}B.
```

Comment:
```suggestion
Overall performance across data types for 84 genes is summarized in Figure {@fig:methylation}B.
```
I'm not sure you need this line. I think you are better off deleting it and referring to this figure in the line below.